lib/node_modules/@stdlib/math/base/special/sici/src/main.c

```diff
@@ -0,0 +1,384 @@
+
+// BEGIN: polyval_gd4
+
+/**
+* Evaluates a polynomial.
+*
+* ## Notes
+*
+* -   The implementation uses [Horner's rule][horners-method] for efficient computation.
+*
+* [horners-method]: https://en.wikipedia.org/wiki/Horner%27s_method
+*
+* @param x    value at which to evaluate the polynomial
+* @return     evaluated polynomial
+*/
+static double polyval_gd4( const double x ) {
+	return 7.825792189335346e-9 + (x * (0.0000020265918208634397 + (x * (0.0001732210814741771 + (x * (0.006223963454417684 + (x * (0.09887717612776888 + (x * (0.666296701268988 + (x * (1.6440220241335535 + (x * 1.0)))))))))))));
+}
+
+// END: polyval_gd4
+
+// BEGIN: polyval_gn8
+
+/**
+* Evaluates a polynomial.
+*
+* ## Notes
+*
+* -   The implementation uses [Horner's rule][horners-method] for efficient computation.
+*
+* [horners-method]: https://en.wikipedia.org/wiki/Horner%27s_method
+*
+* @param x    value at which to evaluate the polynomial
+* @return     evaluated polynomial
+*/
+static double polyval_gn8( const double x ) {
+	return 3.1404009894636335e-15 + (x * (3.859459254302766e-12 + (x * (1.7040445278204452e-9 + (x * (3.471311670841167e-7 + (x * (0.000034894116550227946 + (x * (0.001717182390523479 + (x * (0.03848787676499743 + (x * (0.33041097930563207 + (x * 0.6973599534432762)))))))))))))));
+}
+
+// END: polyval_gn8
+
+// BEGIN: polyval_gd8
+
+/**
+* Evaluates a polynomial.
+*
+* ## Notes
+*
+* -   The implementation uses [Horner's rule][horners-method] for efficient computation.
+*
+* [horners-method]: https://en.wikipedia.org/wiki/Horner%27s_method
+*
+* @param x    value at which to evaluate the polynomial
+* @return     evaluated polynomial
+*/
+static double polyval_gd8( const double x ) {
+	return 3.1404009894636335e-15 + (x * (3.878301660239547e-12 + (x * (1.7269374896631615e-9 + (x * (3.5704322344374083e-7 + (x * (0.00003684755044425611 + (x * (0.0019028442667439953 + (x * (0.04679131942596258 + (x * (0.48785225869530496 + (x * (1.6854889881101165 + (x * 1.0)))))))))))))))));
+}
+
+// END: polyval_gd8
+
+/* End auto-generated functions. */
+
+/**
+* Computes the sine and cosine integrals.
+*
+* ## Method
+*
+* -   The integrals are approximated by rational functions.
+*
+* -   For \\( x > 8 \\), auxiliary functions \\( f(x) \\) and \\( g(x) \\) are employed such that
+*
+*     ```tex
+*     \operatorname{Ci}(x) = f(x) \sin(x) - g(x) \cos(x) \\
+*     \operatorname{Si}(x) = \pi/2 - f(x) \cos(x) - g(x) \sin(x)
+*     ```
+*
+* ## Notes
+*
+* -   Absolute error on test interval \\( \[0,50\] \\), except relative when greater than \\( 1 \\):
+*
+*     | arithmetic | function    | # trials | peak    | rms     |
+*     |:----------:|:-----------:|:--------:|:-------:|:-------:|
+*     | IEEE       | Si          | 30000    | 4.4e-16 | 7.3e-17 |
+*     | IEEE       | Ci          | 30000    | 6.9e-16 | 5.1e-17 |
+*
+* @param x     input value
+* @param si    output pointer for sine integral
+* @param ci    output pointer for cosine integral
```

Comment:
```suggestion
* @param si    destination to store the sine integral
* @param ci    destination to store the cosine integral
```

Following what we have in `sincos`: https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/special/sincos/src/main.c#L97